frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -4,31 +4,75 @@
       <h2 class="card-title">{{ T.loginSignupHeader }}</h2>
     </div>
     <div class="card-body">
-      <form>
+      <form
+        class="needs-validation"
+        :class="{ 'was-validated': wasValidated }"
+        novalidate
+      >
+        <div class="form-group">
+          <label>{{ T.userEditBirthDate }}</label>
+          <omegaup-datepicker
+            v-model="birthDate"
+            name="reg_birthdate"
+            :max="new Date()"
+          ></omegaup-datepicker>
+        </div>
```

Comment:
Just for consistency with the other inputs
```suggestion
        <div class="row justify-content-md-center">
          <div class="col-md-4 col-md-offset-2">
            <div class="form-group">
              <label class="control-label">{{ T.userEditBirthDate }}</label>
              <omegaup-datepicker
                v-model="birthDate"
                name="reg_birthdate"
                :max="new Date()"
              ></omegaup-datepicker>
            </div>
          </div>
        </div>
```